index.html

```diff
@@ -184,7 +184,52 @@ <h2 class="post-tittle">Multi-Post Stories</h2>
           </div>
         </div>
         </div> 
-      </section>
-    
+    </section>
+
+    <section class="contact-container">
+      <div class="frame42">
+        <h2>CONTACT US</h2>
+        <p>
+          If you have an application you are
+          interested in developing, a feature that you
+          need built or a project that needs coding.
+          I’d love to help with it
+        </p>
+      </div>
+      <form method="post" action="https://formspree.io/f/mjvlplnjl">
+        <input type="text" name="text" required placeholder="Text Input">
+        <input type="email" name="email" required id="email"  placeholder="test@gmail.com">
+        <textarea name="text" id="text" maxlength="300">Enter text here</textarea>
+        <button class="bttn" type="button">See Project</button>
+      </form>
+    </section>
+
+    <footer>
+      <hr>
+      <div class="support-item">
+        <div class="social-media">
+          <ul>
+            <li>
+              <a href=""><img class="git-icon" src="images/git-icon.png" alt=" "></a>
+            </li>
+            <li>
+              <a href=""><img class="linkedin-icon" src="images/Linkedin-icon.png" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img class="angelist-icon" src="images/Icon-Angelist.png" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img class="Twitter-icon" src="images/Icon-Twitter.png" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img class="Medium-icon" src="images/Medium-icon.png" alt=""></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+        <p class="footer-text">test@gmail.com</p>
+    </footer>
```

Comment:
- [ ] Please use the correct color for `<hr>` and decrease the margin between social media icons and email addresses and make sure it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166517923-f9469260-3116-479a-840e-8f1fa11c739a.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166517966-e9f12dd4-10dd-418a-9df8-87b186872b0f.png)